packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -513,14 +480,6 @@ export interface IGarbageCollectorConfigs {
 	readonly inactiveTimeoutMs: number;
 	/** Tracks whether GC should run in test mode. In this mode, unreferenced objects are deleted immediately. */
 	readonly testMode: boolean;
-	/**
-	 * Tracks whether GC should run in tombstone mode. In this mode, objects are marked as tombstones as a step along the
-	 * way before they are fully deleted.
-	 * In interactive (non-summarizer) clients, tombstone objects behave as if they are deleted, i.e., access to them
-	 * is not allowed. However, these objects can be accessed after referencing them first. It is used as a "warning"
-	 * step before sweep, where objects wrongly marked as unreferenced can be recovered.
-	 */
-	readonly tombstoneMode: boolean;
```

Comment:
So why is this being removed? don't see it mentioned in the PR description, so just curious if this was dead flag?

Reply:
I missed adding it to the description. I added it now. Basically, tombstone mode is always on if sweep is on.